admin/controllers/users/views/default/view.php

```diff
@@ -55,6 +57,84 @@
 			<button formaction='<?php echo Config::$uripath;?>/admin/users/action/delete' onclick='return window.confirm("Are you sure?")' class='button is-danger' type='submit'>Delete</button>
 		</div>
 	</h1>
+
+	<div id='content_search_controls' class='flex'>
+
+		<div class="field">
+			<label class="label">Search Name/Email</label>
+			<div class="control">
+				<input value="<?php echo $search; ?>" name="search" form="searchform" class="input" type="text" placeholder="">
+			</div>
+		</div>
+
+		<div class='field'>
+			<label class="label">State</label>
+			<div class='control'>
+				<div class="select">
+					<input type='hidden' name='filters[2][key]' value='state' form='searchform'/>
+					<select name="filters[2][value]" form="searchform">
+						<option value=''>State</option>
+						<option <?php if ($filters['state']==='1') { echo " selected "; }?> value='1'>Published</option>
+						<option <?php if ($filters['state']==='0') { echo " selected "; }?> value='0'>Unpublished</option>
+						<option <?php if ($filters['state']==='-1') { echo " selected "; }?> value='-1'>Deleted</option>
```

Comment:
i assume this is copy/pasted from the content search, its probably more user friendly if `Published` is changed to `Enabled` or `Active` and `Unpublished` is changed to `Disabled` or something like that. shouldnt make a difference to the code since the value isnt changing, but just a UX thing

Reply:
Done.